OpenUpSA/wazimap-ng-ui
# point filter dialog changes

## Description
* collapsing / expanding the point mapper filter dialog by javascript instead of webflow interactions
* point mapper filter dialog is collapsed by default
* if there are no fields to filter by, we now show a message that's stating no filters available instead of empty dropdowns

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-149

## How is it tested automatically?
added test steps into `__tests__/gui/point_data.feature`

## How should a reviewer test it locally
* select a category from the point mapper
* wait until point markers are created
* confirm that the point mapper filter dialog is collapsed by default
* expand it
* confirm that if there are no fields to filter by, `No filters available for the selected data.` message is shown in the dialog
   -> e.g Higher Education > Universities on YE profile
* confirm that if there are fields to filter by, filtering works as normal
   -> e.g Higher Education > TVET Colleges on YE profile
* confirm that chart filtering, data mapper filtering, point mapper filtering work as expected

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/145889178-3803d46f-6653-4f3a-a1b8-827343d97619.png)


## Changelog

### Added

### Updated
* `src/js/map/point_data/point_filter.js` to use the new clone of the point filter dialog instead of the old one
* `src/js/map/point_data/filter_controller.js` to show/hide the warning message depending on whether there are fields to filter by or not

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
